shared/templates/sysctl/ansible.template

```diff
@@ -8,8 +8,10 @@
 {{% if product in ["sle12","sle15"] %}}
   find:
     paths:
-      - "/etc/sysctl.d/"
       - "/run/sysctl.d/"
+      - "/etc/sysctl.d/"
+      - "/usr/local/lib/sysctl.d/"
+      - "/lib/sysctl.d/"      
```

Comment:
I guess ` /usr/lib/sysctl.d/ ` is missing, in the Bash remediation it is present.

Reply:
Hello @yuumasato 
If you build for sle12/15 and and check ansible remediation file you will find:

    - name: List /etc/sysctl.d/*.conf files
      find:
        paths:
          - /run/sysctl.d/
          - /etc/sysctl.d/
          - /usr/local/lib/sysctl.d/
          - /lib/sysctl.d/
          - /usr/lib/sysctl.d/
        contains: ^[\s]*kernel.randomize_va_space.*$
        patterns: '*.conf'
        file_type: any
      register: find_sysctl_d
      when: ansible_virtualization_type not in ["docker", "lxc", "openvz", "podman", "container"]
      tags:
        - CCE-83300-4
        - DISA-STIG-SLES-15-010550
        - NIST-800-171-3.1.7
        - NIST-800-53-CM-6(a)
        - NIST-800-53-SC-30
        - NIST-800-53-SC-30(2)
        - PCI-DSS-Req-2.2.1
        - PCI-DSSv4-2.2.3
        - disable_strategy
        - low_complexity
        - medium_disruption
        - medium_severity
        - reboot_required
        - sysctl_kernel_randomize_va_space

Have a nice day
Rumen

Reply:
I see, this path comes from the Jinja  conditional below.
Thanks